source/bindings/KBindSystem.hh

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include <vector>
+
+#include "Common.hh"
+#include "egg/core/Disposer.hh"
+#include "egg/core/SceneManager.hh"
+#include "game/system/KPadController.hh"
+#include "game/system/RaceConfig.hh"
+
+namespace EGG {
+class SceneManager;
+}
```

Comment:
If you want to forward declare here, then you should get rid of the `#include` otherwise it defeats the purpose